src/monitor/ManualAgentsValidator.cpp

```diff
@@ -0,0 +1,21 @@
+
+#include "ManualAgentsValidator.hpp"
+#include "AgentInformation.hpp"
+
+
+void ManualAgentsValidator::addNewAgent(const QString& name, const QString& ip, const QString& port)
```

Comment:
This is an option which allow user to add agent without zeroconf service installed or this is for something else?

Reply:
"This is an option which allow user to add agent without zeroconf service installed" yes